caffe2/python/onnx/tests/onnx_backend_test.py

```diff
@@ -22,6 +22,14 @@
                      '|test_mean|test_hardmax'  # Does not support Mean and Hardmax.
                      '|test_cast.*FLOAT16.*'  # Does not support Cast on Float16.
                      '|test_depthtospace.*'  # Does not support DepthToSpace.
+                     '|test_reduce_l1.*'  # Does not support ReduceL1.
+                     '|test_reduce_l2.*'  # Does not support ReduceL2.
+                     '|test_reduce_log_sum.*'  # Does not support ReduceLogSum.
+                     '|test_reduce_max.*'  # Does not support ReduceMax.
```

Comment:
It's good to prioritize this to have better support of exporting `torch.max(tensor, dim=n)`

Reply:
@bddppq yeah, @BIT-silence is working on it right now, I think.